observability/cockpit/how-to/configure-alerts-for-scw-resources.mdx

```diff
@@ -14,14 +14,15 @@ dates:
 
 This page shows you how to configure managed alerts for Scaleway resources using the `Scaleway Metrics` data source.
 
-<Message type="requirement">
-  - You have an account and are logged into the [Scaleway console](https://console.scaleway.com/)
-  - You have [activated your Cockpit](/observability/cockpit/how-to/activate-cockpit/)
-  - You have created Scaleway resources to monitor
-  - You have [created your Grafana credentials](/observability/cockpit/how-to/retrieve-grafana-credentials/) and selected the **Editor** role
-  - You have [activated managed alerts and created at least one contact point](/observability/cockpit/how-to/activate-managed-alerts/)
-  - You are using the **Scaleway Alerting** alert manager in Grafana
-</Message>
+<Macro id="requirements" />
+
+- A Scaleway account logged into the [console](https://console.scaleway.com)
+- An active [Cockpit](/observability/cockpit/how-to/activate-cockpit/)
+- Scaleway resources you can monitor
+- Grafana [credentials](/observability/cockpit/how-to/retrieve-grafana-credentials/) with the **Editor** role
+- Have [activated managed alerts and created at least one contact point](/observability/cockpit/how-to/activate-managed-alerts/)
+- Use the **Scaleway Alerting** alert manager in Grafana
```

Comment:
Don't think this works with the "To complete the actions presented on this page, you must have..." intro,  but not sure what it should be?

Reply:
"Selected the **Scaleway Alerting** alert manager in Grafana"